AndroidAnnotations/androidannotations-core/androidannotations/src/main/java/org/androidannotations/holder/SharedPrefHolder.java

```diff
@@ -118,8 +119,14 @@ private void createEditMethod() {
 	public void createFieldMethod(Class<?> prefFieldHelperClass, IJExpression keyExpression, String fieldName, String fieldHelperMethodName, IJExpression defaultValue,
 			String docComment, String defaultValueStr) {
 		JMethod fieldMethod = generatedClass.method(PUBLIC, prefFieldHelperClass, fieldName);
+
 		if (defaultValueStr != null) {
-			fieldMethod.javadoc().append("<p><b>Defaults to</b>: " + defaultValueStr + "</p>\n");
+			boolean isStringPrefField = "stringField".equals(fieldHelperMethodName);
```

Comment:
Maybe a little bit nicer check would be `prefFieldHelperClass == StringSetPrefField.class`. So we at least do not rely on magic string constants.